Makefile

```diff
@@ -2,7 +2,7 @@
 
 PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
 # VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
-VERSION := "0.0.5"
```

Comment:
Why don't we use the `VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')` instead? When building from a tag, this command returns the current git tag for the repo, or when you build from outside a git tag, it includes the commit hash in the version string. This is extremely helpful as it details exactly which code version a user is running when they report issues.